src/common/servlist.c

```diff
@@ -196,6 +196,12 @@ static const struct defaultserver def[] =
 #endif
 	{0,			"irc.globalgamers.net"},
 
+#ifdef USE_OPENSSL
+	{"hackint", 0, 0, 0, LOGIN_SASL, 0, TRUE},
+	{0,			"irc.hackint.org/+6697"},
```

Comment:
Drop the explicit port (is the default for TLS), and do we really need two different hostnames?

Reply:
Rebased the pr and dropped the secondary hostname and port.